apl-api2/src/main/resources/yaml/apollo-api-v2.yaml

```diff
@@ -623,587 +626,1649 @@ paths:
+          required: true
+          schema:
+            type: string
+            example: 'APL-X5JH-TJKJ-DVGC-5T2V8'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractSpecResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/asr:
+    get:
+      tags:
+        - smc
+        - asr
+      summary: Get the ASR templates list.
+      description: Returns the ASR templates list.
+      operationId: getSmcAsrModules
+      parameters:
+        - name: language
+          in: query
+          description: The language name
+          required: false
+          schema:
+            type: string
+            example: 'js'
+        - name: version
+          in: query
+          description: The language version
+          required: false
+          schema:
+            type: string
+            example: '0.1.1'
+        - name: type
+          in: query
+          description: The module type, allowed values - token, escrow; the default value is token.
+          required: false
+          schema:
+            type: string
+            example: 'token'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ModuleListResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/asr/{module}/init:
+    get:
+      tags:
+        - smc
+        - asr
+      summary: Get the contract constructor of the given ASR module.
+      description: Returns the contract constructur
+      operationId: getAsrModuleInitSpec
+      parameters:
+        - name: module
+          in: path
+          description: The ASR module name
+          required: true
+          schema:
+            type: string
+            example: 'APL20BUY'
+        - name: language
+          in: query
+          description: The language name
+          required: false
+          schema:
+            type: string
+            example: 'js'
+        - name: version
+          in: query
+          description: The language version
+          required: false
+          schema:
+            type: string
+            example: '0.1.1'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContractSpecResponse'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  /@@smc/asr/{module}/src:
+    get:
+      tags:
+        - smc
+        - asr
+      summary: Get the ASR module source.
```

Comment:
I'd add a ' code ' word to 'source' to clarify the API behaviour

Reply:
fixed